src/nodes/EnvironmentalEffects/X3DBackgroundNode.js

```diff
@@ -41,6 +41,8 @@ x3dom.registerNodeType(
 
             /**
              * Color of the ground
+             * Multiple color values are possible.
+             * You have to define 1 color more that you define ground angles.
```

Comment:
"Define 1 more color than the number of ground angles."

Reply:
English is so simple (but difficult for a German) ;-)